samples/Gonda/.gitignore

```diff
@@ -0,0 +1,4 @@
+obj
```

Comment:
Is this file needed?

Reply:
since there is the root level gitignore this can be removed...